packages/toolkit/src/query/react/buildHooks.ts

```diff
@@ -452,14 +452,7 @@ type UseQueryStateDefaultResult<D extends QueryDefinition<any, any, any, any>> =
             Pick<UseQueryStateBaseResult<D>, 'error'>
           >)
       >
-  > & {
-    /**
-     * @deprecated will be removed in the next version
-     * please use the `isLoading`, `isFetching`, `isSuccess`, `isError`
-     * and `isUninitialized` flags instead
-     */
-    status: QueryStatus
-  }
+  >
```

Comment:
I would actually have kept this one in - the warning should discourage people, but it's there anyways and might be useful in edge cases 😅